Source/diablo.h

```diff
@@ -5,6 +5,10 @@
 
 #ifdef USE_SDL1
 #include "sdl2_to_1_2_backports.h"
+#else
+#if SDL_VERSION_ATLEAST(2, 0, 0)
+#include "sdl2_backports.h"

```

Comment:
Any downsides to always loading this for SDL2?

Reply:
No downsides, done

Reply:
Nice that bring it down to just 11 lines of added complexity from my point of view :)